frontend/templates/events/csr.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=yes" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>CSR | Waves 2018</title>
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!-- Bootstrap core CSS -->
+    <link href="/css/bootstrap.min.css" rel="stylesheet">
+    <!-- Material Design Bootstrap -->
+    <link href="/css/mdb.min.css" rel="stylesheet">
+    <!-- Your custom styles (optional) -->
+    <link href="/css/style.css" rel="stylesheet">
+</head>
+
+<body  style="background-image: url('/img/book.jpg'); background-size: 500px 500px; background-repeat: repeat;">
+   <div id="mySidenav" class="sidenav">
+    <a href="/" class="logobtn"><i class="fa fa-home" style="font-size: 25px; padding-top: 1vh;"></i></a>
+    <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+    <a href="/about-page">About</a>
+    <a href="/contact-us">Contact Us</a>
+    <a href="/our-team">Contact List</a>
+    <a href="/sponsors">Sponsors</a>
+    <a href="/event/csr">CSR</a>
+    <p>Event Registration:</p>
+    <a href="/event/smtf">SMTF</a>
+    <a href="/event/irshad">Irshad</a>
+    <a href="/event/inverse">InVerse</a>
+
+
+        <div class="social-sidebar">
+
+                <a class="p-2 fa-lg fb-ic" href="https://www.facebook.com/bitswaves/">
+                      <i class="fa fa-facebook"></i>
+                </a>
+                    <!-- Twitter -->
+                <a class="p-2 fa-lg tw-ic" href="https://twitter.com/bitswaves?lang=en">
+                      <i class="fa fa-twitter"> </i>
+                </a>
+                    <!-- Instagram -->
+                <a class="p-2 fa-lg ins-ic" href="https://www.instagram.com/wavesbits/">
+                      <i class="fa fa-instagram"> </i>
+                </a>
+
+        </div>
+    </div>
+            <!-- Start your project here-->
+            <a href="#" class="nav-but" data-activates="slide-out" onclick="openNav()"><i class="fa fa-bars" style="font-size: 25px;"></i></a>
```

Comment:
Change color of the icon for opening sidebar. Currently its hard to differentiate from the background.